components/canvas/Section/Section.stories.ts

```diff
@@ -0,0 +1,70 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './Section';
+import '../../control/Button/Button';
+import '../../composition/Grid/Grid';
+import '../../composition/Grid/GridItem';
+
+export default {
+  component: 'diamond-section',
+  argTypes: {
+    theme: {
+      control: {
+        type: 'select',
+      },
+      options: ['light', 'medium', 'dark'],
+    },
+    padding: {
+      control: {
+        type: 'select',
+      },
+      options: ['xs', 'sm', 'md', 'lg', 'xl', 'none'],
+    },
+  },
+};
+
+export const Section: StoryObj = {
+  render: ({ padding, theme }) => html`
+    <diamond-section padding="${padding}" class="diamond-theme-${theme}">
+      <diamond-card>
```

Comment:
Why's it got a card in it?

Reply:
Just the h3 and p is enough to show what the component does

Reply:
Got it, was trying to show how it sets the child components with the theme prop 👍